### PR TITLE
Add gostringsr2

### PR DESCRIPTION
**Pull request by wykwit**
https://github.com/CarveSystems/gostringsr2

### CODE REVIEW

File: packages/gostringsr2/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=gostringsr2
+pkgver=1.1.2
+pkgrel=1
+pkgdesc='Extract strings from a Go binary using radare2'
```
Comment (Edu4rdSHL) [inline packages/gostringsr2/PKGBUILD:7]: End point is missing.

File: packages/gostringsr2/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+build() {
+  cd "$pkgname-$pkgver"
+
+  python setup.py build
+}
+
+package() {
+  cd "$pkgname-$pkgver"
+
+  python setup.py install --root="$pkgdir" --optimize=1 --skip-build
```
Comment (Edu4rdSHL) [inline packages/gostringsr2/PKGBUILD:26]: Please add `--prefix=/usr`

**Review by Edu4rdSHL — CHANGES_REQUESTED** [review]
Hello, @wykwit, thank you so much for it. Please update according to my comments and also add the package to `lists/to-release`.

Regards,
Ed

**Comment by wykwit** [comment]
Thank you for feedback @Edu4rdSHL, hopefully it's OK now.

**Review by Edu4rdSHL — APPROVED** [review]
Thank you, it's ok.

**Pull request merged**